### PR TITLE
1498 - logrotate for /var/log/pulp/*.log  

### PR DESCRIPTION
**Pull request by pronix**
add logrotate  for pulp logs [https://pulp.plan.io/issues/1498]
closes #1498

### CODE REVIEW

**Comment by pronix** [comment]
@bowlofeggs i can't add comment to [https://pulp.plan.io/issues/1498]

**Comment by pronix** [comment]
@jeremycline hello. should i change this PR to pulp/packaging repo ?

File: server/etc/default/pulp.logrotate [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/var/log/pulp/*log {
```
Comment (jeremycline) [inline server/etc/default/pulp.logrotate]: It seems historically we've organized the config files in the path they are installed at under ``server/``. To keep with that style, could you move this to ``server/etc/logrotate.d/pulp``?

**Comment by jeremycline** [comment]
@pronix Hi, thanks for the PR. If you put the spec change in a PR against pulp/packaging that would be very helpful. I'm also happy to pull it out and move it before I merge this pull request, so it's up to you. If you do make a PR against pulp/packaging, just make it against the f24 branch and I'll merge it back to all the appropriate distributions. :star2:

**Comment by pronix** [comment]
@jeremycline thanks for review. changes squashed to current PR. for pulp/packagig PR will soon.

**Comment by jeremycline** [comment]
I've merged this as https://github.com/pulp/pulp/commit/6998f44e6ee19faef9633076bd6472f2ce644d47 - I pulled out the spec changes here so they don't slip into a 2.9.z release while we continue to build with our old spec files.

Thanks again for your contribution!

**Pull request closed without merging**